### PR TITLE
update clinod

### PR DESCRIPTION
**Pull request by ZerryNi**
Test passed!
image

### CODE REVIEW

**Comment by ZerryNi** [comment]
@BiocondaBot please add label.

File: recipes/clinod/clinod.py [hunk 1]
```diff
@@ -35,6 +36,21 @@ def java_executable():
...
+        tf.keras.layers.Dense(64, activation='relu'),
+        tf.keras.layers.Dense(1, activation='sigmoid')
+    ])
+    model.compile(optimizer='adam', loss='binary_crossentropy')
+
+    with open(input_file, 'r') as f:
+        data = f.read()
+
+    
+    result = model.predict([[1.0]])
```
Comment (daler) [inline recipes/clinod/clinod.py:51]: What is the goal of this function? Why would it throw away all data and just predict on `[[1.0]]`?

Comment (ZerryNi) [reply recipes/clinod/clinod.py:51]: Hi.Because the original depengdency on snns was too old and couldn't be adapted to the linux-aarch64 platform,we switched to TensorFlow and modified the test statements for a simple test.All my PR modifications were made to adapt the software to the Linux-aarch64 platform.Thank you for helping me with the problem.

Comment (daler) [reply recipes/clinod/clinod.py:51]: Look at your changes though. This does not add aarch64 support, it just breaks the tool. Any input you give the tool will result in a single float value.

Regardless, this is not something that should be changed in the recipe. It needs to be changed in the actual tool itself. Please work with the upstream authors to make sure this change is OK and supported and actually tested.
